Assets/_BForBoss/_Character/Scripts/FirstPersonPlayer.cs

```diff
@@ -33,6 +42,8 @@ public bool IsThirdPerson
             }
         }
 
+        private float SwitchViewDuration => cmBrain.m_DefaultBlend.m_Time;
```

Comment:
Can you also change the blendtime to 0.5?

Reply:
Yeah, maybe in that case I should modify this to use a custom blend instead of relying on the default blend

Reply:
I just pushed a change that simplifies the logic to remove the coroutine and smooth transition, and since that means the model isn't visible during the blend I went ahead and replaced it with a cut